doc/installation.md

```diff
@@ -173,8 +173,81 @@ We also have [prebuilt Kaldi binaries](https://github.com/espnet/espnet/blob/mas
         $ cd <espnet-root>/tools
         $ rm -f activate_python.sh && touch activate_python.sh
         ```
-1. Install ESPnet
 
+
+#### 4. Install ESPnet via `pyproject.toml` (Recommended for ESPnet3)
+
+ESPnet adopts the modern Python packaging standard using `pyproject.toml`.
+You can install the **core dependencies** with:
+
+* Core installation:
+
+    ```sh
+    $ cd <espnet-root>
+    $ pip install -e ".[core]"
+    ```
+
+To install task-specific dependencies, you can add the desired options. For example:
+
+* For ASR:
+
+    ```sh
+    $ cd <espnet-root>
+    $ pip install -e ".[core,task-asr]"
```

Comment:
1. Is it possible to automatically include core when we specify task-*?
2. what is the default behavior without specifying it.
3. I still cannot understand the role of `core`. If it is used as a standalone? If not, I cannot understand why we make it as a package.